diagnostic.md

```diff
@@ -32,23 +35,29 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getters:
+$('selector').val()
+$('selector').text()
 
+Setters:
+$('selector').val(value)
+$('selector').text('some text')
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+The DOM, or Document Object Model, is a model created by the web browser of the html being rendered on the page. It has a tree structure and can be altered by certain javascript libraries, allowing dynamic webpages.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.